### ISSUE TITLE
Test result is not similar as mentioned in paper

### ISSUE THREAD

**Issue by khoatm98** [Terminal-K/MambaMOS#5]
Hi @Terminal-K , Thank you for sharing your work.
When I test on my server with the instruction for test set, I got the result which is not similar to the one on the paper, do you know why?

<img width="1320" height="177" alt="Image" src="https://github.com/user-attachments/assets/0885f75c-9bdc-4505-8cc2-6e60a41b7308" />

**Comment by zhangsongrong** [comment]
@khoatm98 
I have successfully reproduced your MambaMOS implementation and achieved an mIoU of 0.823, consistent with the result reported in your original paper.
I would like to consult you about two issues encountered during my training:
My server is equipped with 72GB system RAM. The system memory keeps accumulating epoch by epoch without automatic release, eventually running out of available memory and getting killed after only a few training rounds. Have you ever run into this continuous memory leakage problem during your reproduction?
When resuming training from saved checkpoint weights, the training loss surges drastically right after restarting. Did you encounter this abnormal loss jump after resuming training as well?
Thanks a lot for your help!

**Comment by khoatm98** [comment]
hi @zhangsongrong, I'm not the author of this paper, but your issue seems like you had set multiple core in pytorch and each core try to fetch an epoch, which then causes OOM. You might want to try switching to single core first to confirm this point.
